### ISSUE TITLE
install calls the wrong binary

### ISSUE THREAD

**Issue by joshuacox** [micahflee/torbrowser-launcher#187]
just launched and it failed as it was looking for `start-tor-browser` but looking in that directory it is now called `start-tor-browser.desktop`

```
Running /home/bob/.local/share/torbrowser/tbb/x86_64/tor-browser_en-US/start-tor-browser
Unhandled Error
Traceback (most recent call last):
  File "/usr/lib/python2.7/dist-packages/twisted/web/_newclient.py", line 911, in dispatcher
    return func(*args, **kwargs)
  File "/usr/lib/python2.7/dist-packages/twisted/web/_newclient.py", line 1462, in _finishResponse_WAITING
    self._giveUp(Failure(reason))
  File "/usr/lib/python2.7/dist-packages/twisted/web/_newclient.py", line 1515, in _giveUp
    self._disconnectParser(reason)
  File "/usr/lib/python2.7/dist-packages/twisted/web/_newclient.py", line 1503, in _disconnectParser
    parser.connectionLost(reason)
--- <exception caught here> ---
  File "/usr/lib/python2.7/dist-packages/twisted/web/_newclient.py", line 532, in connectionLost
    self.response._bodyDataFinished()
  File "/usr/lib/python2.7/dist-packages/twisted/web/_newclient.py", line 911, in dispatcher
    return func(*args, **kwargs)
  File "/usr/lib/python2.7/dist-packages/twisted/web/_newclient.py", line 1154, in _bodyDataFinished_CONNECTED
    self._bodyProtocol.connectionLost(reason)
  File "/usr/lib/python2.7/dist-packages/torbrowser_launcher/launcher.py", line 378, in connectionLost
    self.all_done(reason)
  File "/usr/lib/python2.7/dist-packages/torbrowser_launcher/launcher.py", line 395, in response_finished
    self.run_task()
  File "/usr/lib/python2.7/dist-packages/torbrowser_launcher/launcher.py", line 332, in run_task
    self.verify()
  File "/usr/lib/python2.7/dist-packages/torbrowser_launcher/launcher.py", line 565, in verify
    self.run_task()
  File "/usr/lib/python2.7/dist-packages/torbrowser_launcher/launcher.py", line 336, in run_task
    self.extract()
  File "/usr/lib/python2.7/dist-packages/torbrowser_launcher/launcher.py", line 609, in extract
    self.run_task()
  File "/usr/lib/python2.7/dist-packages/torbrowser_launcher/launcher.py", line 340, in run_task
    self.run()
  File "/usr/lib/python2.7/dist-packages/torbrowser_launcher/launcher.py", line 640, in run
    subprocess.call([self.common.paths['tbb']['start']])
  File "/usr/lib/python2.7/subprocess.py", line 522, in call
    return Popen(*popenargs, **kwargs).wait()
  File "/usr/lib/python2.7/subprocess.py", line 710, in __init__
    errread, errwrite)
  File "/usr/lib/python2.7/subprocess.py", line 1335, in _execute_child
    raise child_exception
exceptions.OSError: [Errno 2] No such file or directory

 % ls /home/bob/.local/share/torbrowser/tbb/x86_64/tor-browser_en-US/start-tor-browser                                                                                                                     ~ stealth
ls: cannot access /home/bob/.local/share/torbrowser/tbb/x86_64/tor-browser_en-US/start-tor-browser: No such file or directory
 % ls /home/bob/.local/share/torbrowser/tbb/x86_64/tor-browser_en-US/start-tor-browser.desktop                                                                                                             ~ stealth
Browser/                    start-tor-browser.desktop*
```

**Comment by Balliver** [comment]
I am having the same problem, I noticed it when the torbrowser-launcher was re-downloading the torbrowser every time I would launch it. running it in a shell shows me the same errors as Joshua. I'm running Debian (Stable) with version 4.5.3 of the torbrowser-launcher.

**Comment by graphiclunarkid** [comment]
Same problem here.

torbrowser-launcher is trying to launch:

`/home/user/.local/share/torbrowser/tbb/x86_64/tor-browser_en-US/start-tor-browser`

Whereas the path now seems to be:

`/home/user/.local/share/torbrowser/tbb/x86_64/tor-browser_en-US/Browser/start-tor-browser`

Here's my traceback:

```
Running /home/user/.local/share/torbrowser/tbb/x86_64/tor-browser_en-US/start-tor-browser
Unhandled Error
Traceback (most recent call last):
  File "/usr/lib/python2.7/dist-packages/twisted/web/_newclient.py", line 911, in dispatcher
    return func(*args, **kwargs)
  File "/usr/lib/python2.7/dist-packages/twisted/web/_newclient.py", line 1462, in _finishResponse_WAITING
    self._giveUp(Failure(reason))
  File "/usr/lib/python2.7/dist-packages/twisted/web/_newclient.py", line 1515, in _giveUp
    self._disconnectParser(reason)
  File "/usr/lib/python2.7/dist-packages/twisted/web/_newclient.py", line 1503, in _disconnectParser
    parser.connectionLost(reason)
--- <exception caught here> ---
  File "/usr/lib/python2.7/dist-packages/twisted/web/_newclient.py", line 532, in connectionLost
    self.response._bodyDataFinished()
  File "/usr/lib/python2.7/dist-packages/twisted/web/_newclient.py", line 911, in dispatcher
    return func(*args, **kwargs)
  File "/usr/lib/python2.7/dist-packages/twisted/web/_newclient.py", line 1154, in _bodyDataFinished_CONNECTED
    self._bodyProtocol.connectionLost(reason)
  File "/usr/lib/python2.7/dist-packages/torbrowser_launcher/launcher.py", line 378, in connectionLost
    self.all_done(reason)
  File "/usr/lib/python2.7/dist-packages/torbrowser_launcher/launcher.py", line 395, in response_finished
    self.run_task()
  File "/usr/lib/python2.7/dist-packages/torbrowser_launcher/launcher.py", line 332, in run_task
    self.verify()
  File "/usr/lib/python2.7/dist-packages/torbrowser_launcher/launcher.py", line 565, in verify
    self.run_task()
  File "/usr/lib/python2.7/dist-packages/torbrowser_launcher/launcher.py", line 336, in run_task
    self.extract()
  File "/usr/lib/python2.7/dist-packages/torbrowser_launcher/launcher.py", line 609, in extract
    self.run_task()
  File "/usr/lib/python2.7/dist-packages/torbrowser_launcher/launcher.py", line 340, in run_task
    self.run()
  File "/usr/lib/python2.7/dist-packages/torbrowser_launcher/launcher.py", line 640, in run
    subprocess.call([self.common.paths['tbb']['start']])
  File "/usr/lib/python2.7/subprocess.py", line 522, in call
    return Popen(*popenargs, **kwargs).wait()
  File "/usr/lib/python2.7/subprocess.py", line 710, in __init__
    errread, errwrite)
  File "/usr/lib/python2.7/subprocess.py", line 1335, in _execute_child
    raise child_exception
exceptions.OSError: [Errno 2] No such file or directory
```

**Comment by graphiclunarkid** [comment]
I am an idiot. I think this bug was fixed in commit 5f833d73290bd3623bf22caffaed599381d454d9.

I'm running Debian Jesse and the version of torbrowser-launcher in their repositories is 0.1.9-1+deb8u1, in which this bug exists, however I've just built v0.2.0 from source and the bug is not present.

Sorry for not testing the latest version before reporting.

**Comment by micahflee** [comment]
No problem!
